.github/workflows/build.yml

```diff
@@ -0,0 +1,194 @@
+          echo "Base ref: ${{ github.base_ref }}"
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'zulu'
+
+      - name: Get the current PR number
+        uses: jwalton/gh-find-current-pr@v1
+        id: current-pr
+        with:
+          state: open
+
+      - name: Calculate version number
+        id: version
+        run: |-
+          POM_VERSION=$(./mvnw org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dtycho.mode=maven -DskipModules=true -Dexpression=project.version -q -DforceStdout)
+          if [[ "${{ github.base_ref }}" == "master" ]]; then
+            BASE_VERSION=${POM_VERSION}
+            VERSION_NUMBER=${BASE_VERSION}            
+          else
+            BASE_VERSION=$(echo "${POM_VERSION}" | cut -d'-' -f 1)
+            TAG_NAME=$(echo "${{ steps.context.outputs.branch }}" | cut -d ' ' -f2 | tr '#\/\.-' '_')
+            VERSION_NUMBER=${BASE_VERSION}.$(date +%Y%m%d_%H%M%S)_${{ steps.context.outputs.shortCommitId }}_${TAG_NAME//[(\)]}
+          fi
+          echo "Version from POM: ${POM_VERSION}"
+          echo "Base version from POM: ${BASE_VERSION}"
+          echo "Building version: ${VERSION_NUMBER}"
+          echo "::set-output name=version::${VERSION_NUMBER}"
+
+      - name: Remove settings.xml
+        run: rm $HOME/.m2/settings.xml || true
+
+      - name: Setup maven settings.xml
+        uses: whelk-io/maven-settings-xml-action@v20
+        with:
+          servers: >
+            [
+                {
+                    "id": "judong-nexus-mirror",
+                    "username": "${{ secrets.JUDONG_NEXUS_USERNAME }}",
+                    "password": "${{ secrets.JUDONG_NEXUS_PASSWORD }}"
+                },
+                {
+                    "id": "judong-nexus-distribution",
+                    "username": "${{ secrets.JUDONG_NEXUS_USERNAME }}",
+                    "password": "${{ secrets.JUDONG_NEXUS_PASSWORD }}"
+                }
+            ]
+          mirrors: >
+            [
+                {
+                    "id": "judong-nexus-mirror",
+                    "mirrorOf": "*",
+                    "url": "https://nexus.judo.technology/repository/maven-judong/"
+                 }
+            ]
+
+      # Phase 1 - Build and deploy to judong nexus
+      - name: Build with Maven (build, test)
+        run: |-
+          ./mvnw -B -Dstyle.color=always \
+          -Drevision=${{ steps.version.outputs.version }} \
+          -Psign-artifacts \
+          -Prelease-judong \
+          deploy
+
+      # For maven central repository release
+      - name: Deploy with Maven (central)
+        if: ${{ always() && contains(github.head_ref, 'release') && job.status == 'success' }}
+        run: |-
+          ./mvnw -B -Dstyle.color=always \
+          -Drevision=${{ steps.version.outputs.version }} \
+          -DdeployOnly \
+          -P"release-central,sign-artifacts,-verify-feature" \
```

Comment:
`-verify-feature`, do we need the leading `-`?

Reply:
minus(-) means ignoring that profile. We want to ignore verify-feature profile.